Sources/ChainedMocks.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ChainedMocks.swift
+//  Mocker
+//
+//  Created by Aleksey Kuznetsov on 20.09.2021.
+//  Copyright © 2021 WeTransfer. All rights reserved.
+//
+
+import Foundation
+
+/// Chained mocks representation which can be used for mocking data requests where
+/// order is important or same requests can return different data.
+public class ChainedMocks {
```

Comment:
<!--
  0 failure: 
  1 warning:  Consider using fi...
  
  
  DangerID: danger-id-Danger;
  File: Sources/ChainedMocks.swift;
  Line: 13;
-->

- :warning: Consider using final for this class or use a struct (final_class)